From 4106ee50f9bc3c03dd41087150055fe5029fb3b0 Mon Sep 17 00:00:00 2001
From: Marc Hoersken <info@marc-hoersken.de>
Date: Sun, 12 Apr 2015 10:15:35 +0200
Subject: [PATCH 6/8] Merge r1533765, r1621419, r1638159, r1638188, r1601603,
 r1638412, r1418763 from trunk

    ssl: Axe needless string duplication in setup for call to apr_proc_create()
    Fix sscanf format spotted by cppcheck
    fix indent.
    fix style
    Private function doesn't need ap_ prefix.
    tab vs space
    rename variables: s should be the server_rec

Submitted by: trawick, jailletc36, jailletc36, jailletc36, takashi, jailletc36, sf
Reviewed by: jailletc36, ylavic, covener
Backported by: jailletc36

git-svn-id: https://svn.apache.org/repos/asf/httpd/httpd/branches/2.4.x@1642403 13f79535-47bb-0310-9956-ffa450edef68

# Conflicts:
#	STATUS
#	modules/dav/fs/repos.c
#	modules/ssl/ssl_engine_init.c
#	modules/ssl/ssl_engine_io.c
#	server/protocol.c
---
 modules/proxy/mod_proxy_wstunnel.c | 4 ++--
 modules/proxy/proxy_util.c         | 6 +++---
 2 files changed, 5 insertions(+), 5 deletions(-)

diff --git a/modules/proxy/mod_proxy_wstunnel.c b/modules/proxy/mod_proxy_wstunnel.c
index f924e8c..819864d 100644
--- a/modules/proxy/mod_proxy_wstunnel.c
+++ b/modules/proxy/mod_proxy_wstunnel.c
@@ -167,7 +167,7 @@ static void remove_reqtimeout(ap_filter_t *next)
 /*
  * process the request and write the response.
  */
-static int ap_proxy_wstunnel_request(apr_pool_t *p, request_rec *r,
+static int proxy_wstunnel_request(apr_pool_t *p, request_rec *r,
                                 proxy_conn_rec *conn,
                                 proxy_worker *worker,
                                 proxy_server_conf *conf,
@@ -394,7 +394,7 @@ static int proxy_wstunnel_handler(request_rec *r, proxy_worker *worker,
 
 
         /* Step Three: Process the Request */
-        status = ap_proxy_wstunnel_request(p, r, backend, worker, conf, uri, locurl,
+        status = proxy_wstunnel_request(p, r, backend, worker, conf, uri, locurl,
                                       server_portstr);
         break;
     }
diff --git a/modules/proxy/proxy_util.c b/modules/proxy/proxy_util.c
index 1ff0002..c0666d8 100644
--- a/modules/proxy/proxy_util.c
+++ b/modules/proxy/proxy_util.c
@@ -2333,7 +2333,7 @@ static apr_status_t send_http_connect(proxy_conn_rec *backend,
 
     /* Check for HTTP_OK response status */
     if (status == APR_SUCCESS) {
-        int major, minor;
+        unsigned int major, minor;
         /* Only scan for three character status code */
         char code_str[4];
 
@@ -2351,7 +2351,7 @@ static apr_status_t send_http_connect(proxy_conn_rec *backend,
                 ap_log_error(APLOG_MARK, APLOG_ERR, 0, s,
                              "send_http_connect: the forward proxy returned code is '%s'",
                              code_str);
-            status = APR_INCOMPLETE;
+                status = APR_INCOMPLETE;
             }
         }
     }
@@ -2947,7 +2947,7 @@ PROXY_DECLARE(int) ap_proxy_create_hdrbrgd(apr_pool_t *p,
 
         /* for sub-requests, ignore freshness/expiry headers */
         if (r->main) {
-            if (    !strcasecmp(headers_in[counter].key, "If-Match")
+            if (   !strcasecmp(headers_in[counter].key, "If-Match")
                 || !strcasecmp(headers_in[counter].key, "If-Modified-Since")
                 || !strcasecmp(headers_in[counter].key, "If-Range")
                 || !strcasecmp(headers_in[counter].key, "If-Unmodified-Since")
-- 
2.3.5

